<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="gerer_f_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="gerer_f">
     <attribute name="title">
      <string>Gerer fournisseur</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>9</y>
        <width>701</width>
        <height>361</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter un fournisseur</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Tel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>240</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Mail</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_four">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_adresse_four">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel_four">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>170</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mail_four">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>240</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>190</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_four">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher un fournisseur</string>
       </attribute>
       <widget class="QTableView" name="tabfournisseur">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>11</y>
          <width>411</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>30</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_four_2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>80</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>80</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>150</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_four_4">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>150</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rechercher_id">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>120</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher par id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>210</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_four_4">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>210</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>290</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_adresse_four_4">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>290</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rechercher_nom">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>183</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher par nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rechercher_adresse">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>252</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher par adresse</string>
        </property>
       </widget>
       <widget class="QTableView" name="four_trouve">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>421</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Modifier un fournisseur</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Tel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Mail</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_four_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_adresse_four_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel_four_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mail_four_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>240</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>434</x>
          <y>162</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_four_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="gerer_p">
     <attribute name="title">
      <string>Gerer produit</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>9</y>
        <width>691</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Ajouter un produit</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>id fournisseur</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantite</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_prod">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_prod">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix_prod">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_quantite_prod">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_prod">
        <property name="geometry">
         <rect>
          <x>404</x>
          <y>182</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Afficher un produit</string>
       </attribute>
       <widget class="QTableView" name="tabproduit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>381</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>72</y>
          <width>47</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_prod">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>20</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>160</y>
          <width>47</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_imprimer">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>110</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioB_tri_id">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>190</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Trie par id</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioB_tri_nom">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>220</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Trie par nom</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioB_tri_prix">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>260</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Trie par prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_prod_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>70</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_prod_4">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>160</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Modifier un produit</string>
       </attribute>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Id</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_prod_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>prix</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantite</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_prod_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix_prod_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_quantite_prod_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_prod">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>170</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
